torch/csrc/api/include/torch/nn/options/loss.h

```diff
@@ -18,6 +18,21 @@ struct TORCH_API L1LossOptions {
 
 // ============================================================================
 
+/// Options for a KLDiv loss module.
+using KLDivLossOptions = L1LossOptions;
+
+// ============================================================================
+
+/// Options for a MSE loss module.
+using MSELossOptions = L1LossOptions;
+
+// ============================================================================
+
+/// Options for a BCE loss module.
+using BCELossOptions = L1LossOptions;
```

Comment:
I think `BCELossOptions` can take `weight` and `reduction` as constructor args, so it's not strictly equivalent to `L1LossOptions`.